### DISCUSSION TITLE
How to hide a Move Button ?

### DISCUSSION THREAD

**Discussion by paingchan** [awcodes/filament-table-repeater#111] [Q&A] [answered]
<img width="803" alt="Screenshot 2024-04-02 at 2 14 23 AM" src="https://github.com/awcodes/filament-table-repeater/assets/54994337/a715b523-22e1-4bde-b90e-8aa02bebc363">

**Comment by awcodes** [comment, marked as answer]
item actions work the same way they do on Filament's native repeater field.

**Reply by paingchan** [reply]
Thanks you , using ->reorderableWithDragAndDrop(false) work now.
